cli/js/lib.deno.ns.d.ts

```diff
@@ -32,12 +33,16 @@ declare namespace Deno {
    * when `Deno.runTests` is used */
   export function test(name: string, fn: TestFunction): void;
 
+  enum TestStatus {
+    Passed = "passed",
+    Failed = "failed",
+    Skipped = "skipped"
+  }
+
   interface TestResult {
-    passed: boolean;
     name: string;
-    skipped: boolean;
-    hasRun: boolean;
-    duration: number;
+    status: TestStatus;
+    duration?: number;
```

Comment:
not optional anymore

Reply:
It's not there for `skip = true`, or does Rust do it?

Reply:
Right... Rust doesn't show duration of tests at all